charts/astronomer/templates/houston/helm-hooks/houston-upgrade-deployments-job.yaml

```diff
@@ -53,11 +53,6 @@ spec:
           resources:
 {{ toYaml .Values.houston.resources | indent 12 }}
           env:
-            - name: DATABASE_URL
-              valueFrom:
-                secretKeyRef:
-                  name: astronomer-bootstrap
-                  key: connection
```

Comment:
https://github.com/astronomer/astronomer/blob/091dd67965b316d8aff4c57ef355bcaee1ff4a58/charts/astronomer/templates/_helpers.yaml#L203-L207

@pgvishnuram and here as you can see we use `name: astronomer-bootstrap` but in `_helpers.yaml` we see `name: {{ template "houston.backendSecret" . }} `